lib/screens/explore_screens/regions_screens/regions_main/interactor/viewmodel/region_state.dart

```diff
@@ -20,6 +20,8 @@ class RegionState extends Equatable {
   @override
   List<Object?> get props => [pageCommand, pageState, region, isBrowseView, userType];
 
+  bool get canCreateAnEvent => !isBrowseView || userType == TypeOfUsers.admin;
```

Comment:
should this be !isBrowseView && userType == TypeOfUsers.admin;

??

Reply:
Since a creator is not a firebase member we need to add this check.

Reply:
but the only one allow to create events is the admin right?

Reply:
Right let me check seems to be an OR instead a AND operator.

Reply:
Oh wait i set an OR.
Are you aking if both roles have the ability to create events?